app/controllers/aircrafts_controller.rb

```diff
@@ -8,6 +8,7 @@ def index
 
   # GET /aircrafts/1 or /aircrafts/1.json
   def show
+    authorize(Aircraft)
```

Comment:
```suggestion
    authorize(@aircraft)
```
add this to new, edit, update and destroy